apps/jobs/sender.py

```diff
@@ -83,6 +84,16 @@ def publish_submission_message(message):
     queue_name = challenge.queue
     slack_url = challenge.slack_webhook_url
     queue = get_or_create_sqs_queue(queue_name)
+    # increase counter for submission pushed into queue
+    submission_pk = message["submission_pk"]
+    statsd.gauge(
+        NUM_SUBMISSIONS_IN_QUEUE,
+        1,
+        tags=[
+            "queue_name:%s" % queue_name,
+            "submission_pk:%s" % submission_pk,
+        ],
+    )
```

Comment:
call for `statsd.gauge` should be in a util method. You should pass `metric_name` and `tags` to this util method